docs/optics.md

```diff
@@ -3,20 +3,28 @@ id: optics
 title: Tofu optics
 ---
 
+## Installation
+`"ru.tinkoff" %% "tofu" % tofu-version`  
+or as standalone dependencies:   
+`"ru.tinkoff" %% "tofu-optics-core" % tofu-version`  
+`"ru.tinkoff" %% "tofu-optics-macro" % tofu-version`  
+`"ru.tinkoff" %% "tofu-optics-interop" % tofu-version`  
+ 
+## Functionality
 Tofu contains its own optics library with some notable features:
 - subtyping: You don't need to convert more powerful optic to weaker one
 - unified composition operator
 
 Hierarchy
 ---------
 
-[![Optics Hierarchy](./optics-hierarchy.png)]
+<img src="/tofu/docs_img/optics-hierarchy.png" height="700">
 
 Intuition
 ---------
 
-If You ever used [lens](https://github.com/ekmett/lens) optics' names will be quite familiar to You,
-but for a [Monocle](https://github.com/julien-truffaut/Monocle) user Tofu's naming may be confusing. The following table may clear things up:
+If you ever used [lens](https://github.com/ekmett/lens) optics' names will be quite familiar to you,
```

Comment:
Pronouns are written from the capital letter in english, this must be reverted

Reply:
https://www.dictionary.com/e/when-to-capitalize-words/
"Pronouns are words that replace nouns. I, you, and me are all examples of pronouns. While you and me are usually lowercase, the pronoun I should always be capitalized, regardless of where it appears in a sentence."

Reply:
Roger that